lib/request-base.js

```diff
@@ -164,6 +164,10 @@ exports.field = function(name, val) {
   if (!this._formData) {
     var FormData = require('form-data'); // browserify compatible. May throw if FormData is not supported natively.
     this._formData = new FormData();
+    this._formData.on('error', function(err) {
```

Comment:
This may not work in the browser, where we use native `FormData` without `on` methods